crates/ide-completion/src/render.rs

```diff
@@ -349,6 +349,23 @@ fn render_resolution_path(
 
         path_ref_match(completion, path_ctx, &ty, &mut item);
     };
+
+    if let ScopeDef::ModuleDef(ModuleDef::Adt(adt)) = resolution {
```

Comment:
@Veykril is this the sort of thing you had in mind?

Reply:
Yes, that should be what I had in mind